lib/datadog/profiling/agent_settings_resolver.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+require_relative '../core/configuration/agent_settings_resolver'
+
+module Datadog
+  module Profiling
+    # Any profiling specific agent settings should be added here rather than in the core resolver
```

Comment:
I think this description can be more "plain", simply explaining what this class does, and not necessarily referencing its history, something like "this class encapsulates how the profiler communicates with the Datadog agent."

Reply:
That's a fair point. I'll update it.